e2e_tests/README.md

```diff
@@ -0,0 +1,65 @@
+# e2elive
+
+An integration testing tool. It leverages an artifact generated by the go-algorand integration tests to ensure all features are exercised.
+
+# Usage
+
+A python virtual environment is highly encouraged.
+
+## Setup environment with venv
+```sh
+cd e2e_tests
+
+# create venv dir, I believe you could specify the version of python here
+python3 -m venv venv_dir
+
+# This command depends on your shell
+source venv_dir/bin/activate.fish
+
+# Install 'e2elive' and 'e2econduit'
+python3 -m pip install .
+```
+
+## Running locally
+
+
+### e2elive
+
+This tests runs the `algorand-indexer` binary. Depending on what you're doing, you may want to use a different value for `--s3-source-net`. After the test finishes, a few simple queries are executed.
+
+This command requires a postgres database, you can run one in docker:
+
+```sh
+docker run --rm -it --name some-postgres -p 5555:5432 -e POSTGRES_PASSWORD=pgpass -e POSTGRES_USER=pguser -e POSTGRES_DB=mydb postgres
+```
+
+Running e2elive
+
+**Note**: you will need to restart this container between tests, otherwise the database will already be initialized. Being out of sync with the data directory may cause unexpected behavior and failing tests.
+```sh
+e2elive --connection-string "host=127.0.0.1 port=5555 user=pguser password=pgpass dbname=mydb" --s3-source-net "fafa8862/rel-nightly" --indexer-bin ../cmd/algorand-indexer/algorand-indexer --indexer-port 9890
+```
+
+### e2econduit
+
+This runs a pipeline with conduit. The database is mocked so a real DB is not required.
```

Comment:
The db is not mocked, it's the same as the indexer e2e tests ( a postegresql container), it just manages that container's lifecycle for you instead of you having to spin it up either via docker-compose or manually.

Reply:
Cool, I missed that. So the only difference is that it doesn't call `e2equeries` at the end?

Reply:
updated

Reply:
> So the only difference is that it doesn't call e2equeries at the end? 

Yeah doesn't introspect into the db to make all of the queries that we do in e2elive, but it plays al of the same data against the indexer conduit pipeline and ensures we hit the expected round in the pipeline w/o error.

The hope (expectation) is that we'll extend the set of test cases though as we get new plugins. In that case `e2econduit` would run the entire set of test cases. Potentially it will be useful in the future to add a CLI command to be able to specify a single test scenario to run instead of all of them.